ruby-challenges.rb

```diff
@@ -30,3 +91,52 @@
 # Expected output example: my_bike.pedal_faster(18) => 28
 # Expected output example: my_bike.brake(5) => 23
 # Expected output example: my_bike.brake(25) => 0
+
+#Pseudocode
+#Create a class called 'Bike'
+#Create setter/getter with 'attr_accessor', add all keys
+#Create setter with initialze and add all valid parameters
+#Create key instances '@model, @wheels, @current_speed'
+#Create method 'bike_info' and return a string interpolation with key instances
+#Create method 'increase_speed' with 'mph' as a parameter. Add 'mph' value to @current_speed
+#Create method 'decrease_speed' with 'mph' as a parameter. Add 'mph' value to @current_speed. Speed cannot go negative since @current_speed is defaulted to 0.
+
+class Bike
+    attr_accessor :model, :wheels, :current_speed
+    
+    def initialize (model)
+        @model = model
+        @wheels = 2
+        @current_speed = 0
+    end
+
+    def bike_info
+        "This #{@model} has #{@wheels} wheels and is going #{@current_speed} mph"
+    end
+
+    def increase_speed mph
+       @current_speed += mph
+    end
+
+    def decrease_speed mph
+        @current_speed -= mph
+    end
+end
+
```

Comment:
While `@current_speed` is defaulted to 0, that doesn't mean it cannot be negative. As it is written here, if you were to keep decreasing speed, the current_speed would go negative. Other than that, looks great!